apps/costcontrol/js/utils/formatting.js

```diff
@@ -208,6 +207,34 @@ var Formatting = (function() {
     formatTimeHTML: formatTimeHTML,
 
     // Given the API information compute the human friendly minutes
-    computeTelephonyMinutes: computeTelephonyMinutes
+    computeTelephonyMinutes: computeTelephonyMinutes,
+
+    formats: {}
```

Comment:
deal! :) Yeah, I misunderstood you, but I think the outcome is good. We are caching what should be cached :)